### ISSUE TITLE
Auto-Logout Page refresh 

### ISSUE THREAD

**Issue by DCWNZ** [nuxsmin/sysPass#947]
Is it possible to add the feature that when the Auto-Logout timer is reached the session is automatically redirected to the sign-in page? Currently the session sits at the last page page looked at.

**Comment by nuxsmin** [comment]
Hello, this feature could be implemented in next versions. I haven't done it yet because it need to set a timer which polls every X seconds for a response, so it would add some extra work to both sides (client an server).

Regards

**Comment by DCWNZ** [comment]
Would be great to have available so pages dont get left showing whatever a lazy user looked at last :-)

**Comment by vmario89** [comment]
I also thought about that. and it would be cool

Same behaviour is if you logout from sysPass. The login mask only will be shown again when you click on that information bubble which tells you that you logged out successfully. This message should disappear automatically:
![grafik](https://user-images.githubusercontent.com/5631071/49339595-d2ca3680-f634-11e8-97e0-5bb66835e19b.png)

**Comment by Weptun** [comment]
Yes, please implement! It often happens that we start modifying the filters or type some search text, only to be logged out after we have finished typing.
